source/fuzz/fuzzer_pass_donate_modules.cpp

```diff
@@ -646,12 +646,13 @@ void FuzzerPassDonateModules::HandleFunctions(
           }
         });
 
-    if (make_livesafe) {
-      // Make the function livesafe and then add it.
-      AddLivesafeFunction(*function_to_donate, donor_ir_context,
-                          *original_id_to_donated_id, donated_instructions);
-    } else {
-      // Add the function in a non-livesafe manner.
+    // If |make_livesafe| is true, try to add the function in a livesafe manner.
+    // Otherwise, or if an attempt to make the function livesafe has failed,
```

Comment:
`Otherwise, or if` -> `Otherwise, if `

Reply:
The comment was correct here. I meant
> Otherwise (if |make_livesafe| is false), or if ...

It needed some clarification, though. Fixed in bdc10e9.

Reply:
You're right, @Vasniktel.
Sorry for the confusion!